pilot/util/default.cfg

```diff
@@ -94,6 +94,12 @@ timing_file: pilot_timing.json
 
 [Information]
 
+# Path to cache
+cache_dir: /lustre/atlas/proj-shared/csc108/debug/atlas/HPC_pilot_test/queue_cache
```

Comment:
Danila, I guess it's HPC specific, but all pilots are affected in this way, is it expected?